lib/options.py

```diff
@@ -65,134 +75,209 @@ def __init__(self):
                 action="store_true",
                 default=False,
-                help="""Start the server under 'gdbserver'. This option is
-                mutually exclusive with --valgrind, --gdb, --lldb and --strace.
-                Default: false.""")
+                help=format_help(
+                    """
+                    Start the server under 'gdbserver'. This option is mutually
+                    exclusive with --valgrind, --gdb, --lldb and --strace.
+
+                    Default: false.
+                    """))
 
         parser.add_argument(
                 "--lldb",
                 dest="lldb",
                 action="store_true",
                 default=False,
-                help="""Start the server under 'lldb' debugger in detached
-                Screen. This option is mutually exclusive with --valgrind,
-                --gdb, --gdbserver and --strace.
-                Default: false.""")
+                help=format_help(
+                    """
+                    Start the server under 'lldb' debugger in detached Screen.
+                    This option is mutually exclusive with --valgrind, --gdb,
+                    --gdbserver and --strace.
+
+                    Default: false.
+                    """))
 
         parser.add_argument(
                 "--valgrind",
                 dest="valgrind",
                 action="store_true",
                 default=False,
-                help="""Run the server under 'valgrind'. This option is
-                mutually exclusive with --gdb, --gdbserver, --lldb and
-                --strace.
-                Default: false.""")
+                help=format_help(
+                    """
+                    Run the server under 'valgrind'. This option is mutually
+                    exclusive with --gdb, --gdbserver, --lldb and --strace.
+
+                    Default: false.
+                    """))
 
         parser.add_argument(
                 "--strace",
                 dest="strace",
                 action="store_true",
                 default=False,
-                help="""Run the server under 'strace'. This option is mutually
-                exclusive with --valgrind, --gdb, --gdbserver, --lldb and
-                --strace.
-                Default: false.""")
+                help=format_help(
+                    """
+                    Run the server under 'strace'. This option is mutually
+                    exclusive with --valgrind, --gdb, --gdbserver, --lldb and
+                    --strace.
+
+                    Default: false.
+                    """))
 
         parser.add_argument(
                 "--builddir",
                 dest="builddir",
                 default="..",
-                help="""Path to project build directory. Default: ".." """)
+                help=format_help(
+                    """
+                    Path to project build directory.
+
+                    Default: ".."
```

Comment:
1. The dot is missing at the end of the sentence.
2. IMHO, `Default: ../ .` looks better.

But all this is up to you.

Reply:
I would prefer the following:

```diff
diff --git a/lib/options.py b/lib/options.py
index 834ba91..72dd77b 100644
--- a/lib/options.py
+++ b/lib/options.py
@@ -236,7 +236,11 @@ class Options(object):
                     """
                     Path to project build directory.
 
-                    Default: ".."
+                    Beware: A relative path is resolved against the directory,
+                    where all tests reside. **NOT** a current directory of a
+                    parent shell.
+
+                    Default: .. (parent directory).
                     """))
 
         parser.add_argument(
@@ -257,6 +261,10 @@ class Options(object):
                     """
                     Path to data directory.
 
+                    Beware: A relative path is resolved against the directory,
+                    where all tests reside. **NOT** a current directory of a
+                    parent shell.
+
                     Default: var.
                     """))
```

Applied this change and force-pushed.

Reply:
I am ok with it.